jinja2/filters.py

```diff
@@ -445,7 +445,7 @@ def do_urlize(eval_ctx, value, trim_url_limit=None, nofollow=False,
     return rv
 
 
-def do_indent(s, width=4, indentfirst=False):
+def do_indent(s, width=4, indentfirst=False, indentblanklines=False):
```

Comment:
I know @ThiefMaster said it's consistent with `indentfirst`, but I'd rather just break consistency and use the easier to read `indent_blank_lines`.